Dungeoneer/assets/data/entities.dat

```diff
@@ -1,5 +1,10 @@
 {
 	entities:{
+	    "Features": {
```

Comment:
If we are replacing the existing Text entity with the NeoText, we should remove this.